app/controllers/candidate_interface/decisions_controller.rb

```diff
@@ -72,7 +72,7 @@ def confirm_withdrawal_feedback
       if @withdrawal_feedback_form.save(@application_choice)
         flash[:success] = "Your application for #{@application_choice.current_course.name_and_code} at #{@application_choice.provider.name} has been withdrawn"
 
-        redirect_to candidate_interface_application_complete_path
+        redirect_to applications_list_page
```

Comment:
I was not inspired and I didn't had a good name for this. Let me know a better name or if this is a good one. Also this will be deleted when we do the cleanup next recruitment cycle!

Reply:
I think this is ok, it's reasonably clear and like you said it'll be removed as part of the cleanup.